EarTrumpet/UI/Helpers/WindowAnimationLibrary.cs

```diff
@@ -42,7 +42,7 @@ public static void BeginFlyoutEntranceAnimation(Window window, WindowsTaskbar.St
                 Duration = new Duration(TimeSpan.FromMilliseconds(266)),
                 FillBehavior = FillBehavior.Stop,
                 EasingFunction = new ExponentialEase { EasingMode = EasingMode.EaseOut },
-                From = 0.5,
+                From = 0.2,
```

Comment:
This was arbitrary, and chosen because I thought the old fade effect was too subtle. No issues with changing it back to the old value.